bot/src/events/onReactionAdd.ts

```diff
@@ -0,0 +1,7 @@
+import { onReactionAddAction } from "../bot";
+
+const event: onReactionAddAction = async ({ client, reaction, user }) => {
```

Comment:
Pokud nepoužíváš jeden z vybalených prvků z objektu, tak jej nemá cenu vybaloval. Tedy není potřeba aby vůbec byl vybalen.

Tedy stačí

```ts
const event: onReactionAddAction = async ({ reaction, user }) => {
```

Reply:
@slunimara

Reply:
Jdu na to.